src/AspNet.Security.OAuth.Reddit/RedditAuthenticationHandler.cs

```diff
@@ -32,6 +32,10 @@ public RedditAuthenticationHandler([NotNull] HttpClient client)
             request.Headers.Accept.Add(new MediaTypeWithQualityHeaderValue("application/json"));
             request.Headers.Authorization = new AuthenticationHeaderValue("bearer", tokens.AccessToken);
 
+            if (!string.IsNullOrWhiteSpace(Options.UserAgent)) {
+                request.Headers.Add("User-Agent", Options.UserAgent);
```

Comment:
Okay, I'm sold :smile: 

Can you please triple check the added user agent **replaces** the default user agent used by the OAuth2 generic middleware?